moduleframework/common.py

```diff
@@ -322,9 +318,31 @@ def __init__(self, *args, **kwargs):
         self.arch = None
         self.dependencylist = {}
         self.moduledeps = None
+        self.is_it_module = False
         # general use case is to have forwarded services to host (so thats why it is same)
         self.ipaddr = trans_dict["HOSTIPADDR"]
         trans_dict["GUESTARCH"] = self.getArch()
+        self.loadconfig()
+
+    def loadconfig(self):
+        """
+        Load configuration from config.yaml file.
+
+        :return: None
+        """
+        self.config = get_config()
+        doc_name = ['modularity-testing', 'meta-test-family', 'meta-test']
+        if self.config.get('document') not in doc_name:
+            raise ConfigExc("bad yaml file: item (%s)" %
+                            doc_name, self.config.get('document'))
+        if self.config.get('modulemd-url') and get_if_module():
+            self.is_it_module = True
+        if self.is_it_module:
+            self.getModulemdYamlconfig()
+        self.moduleName = sanitize_text(self.config['name'])
+        self.source = self.config.get('source')
+        if not self.is_it_module:
+            trans_dict["GUESTPACKAGER"] = "yum -y"
```

Comment:
Hi, this variable is important, because of actual hack, that it checks more than one thing (`self.config.get('modulemd-url') and get_if_module()`)
Then self.is_it_module is much simplier to use. Actually it is used on more places, and probably it will be used more extensively in code in future. so it is better to have just one variable to indicate if it is module or not, that remember and duplicate the same code like `self.config.get('modulemd-url') and get_if_module()` everywhere.